repository/src/main/resources/db/changelog/db.changelog-rdm-6246.2.xml

```diff
@@ -0,0 +1,48 @@
+<!--
+Migrate Webhook timeouts table to an integer array column on Webhooks.
+
+This reduces database query time ~30% when loading a divorce CaseTypeEntity;
+timeouts are typically few in number so modelling them as a relation carries
+a lot of overhead when we don't need the flexibility of the relational approach.
+
+This change can be done without downtime; the migration is backwards compatible with
+the previous code (webhook_timeouts should be dropped on a follow up change).
+-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="rdm-6246.add-timeouts-column" author="alex.mcausland@hmcts.net">
+        <sql>
+            alter table webhook
+            add column timeouts integer[]
+            /* Disallow null values in the array*/
```

Comment:
I think this should be done at the application level not the db like we do other validations. Check the *Validator classes